### ISSUE TITLE
Installation of the repo caffe version does not link properly

### ISSUE THREAD

**Issue by tanismar** [robotology/himrep#32]
I tried to install Caffe cloning the latest version from the repository, and had an issue with himrep not linking to the proper installed libraries, but rather to the build, which does not have some of the required includes. We tried all sort of things to make it link (env variables, path by hand, etc) to make it link to the installed library, but the issue seems to be in a somehow obscure path variable that is set during caffe installation, which overrides the Caffe_DIR or any other env variable set. 
In summary, we were not able to solve this issue, but the installation worked by checking out the caffe version pointed out on the VVV17 installation guide (https://github.com/vvv-school/vvv17/wiki/How-to-prepare-your-system). 
Point being, thus, that it would be a good idea to link to that version on the caffe installation instructions rather then the up-to-date repository.
